nim_status/client.nim

```diff
@@ -274,6 +274,16 @@ proc addWalletSeed*(self: StatusObject, mnemonic: Mnemonic, name, password,
   except Exception as e:
     return WalletAccountResult.err e.msg
 
+proc addWalletWatchOnly*(self: StatusObject, address: Address,
+  name: string): WalletAccountResult =
+
+  try:
+    return self.storeWalletAccount(name, address, SkPublicKey.none,
```

Comment:
Is `return` necessary here and on L286? It seems like the expressions being of type `WalletAccountResult` should suffice for them to be returned automatically, but maybe that doesn't work in this kind of setup, not sure.

Reply:
It is not, was simply to be explicit for readability's sake.